### PR TITLE
Estimated hypertensive population subtitle

### PR DESCRIPTION
**Pull request by claudiovallejo**
**Story card:** [ch6166](https://app.shortcut.com/simpledotorg/story/6166/add-estimated-hypertensive-population-subtitle-to-district-cards-in-facilities-admin-page)

## Because

Facility group cards in the "Facilities" page will now display the "Estimated hypertensive population" value set for each facility group.

## This addresses

Shows a population value or an "+ Add population" link (when a value hasn't been set)

## Test instructions

1. Go to the "Facilities" admin page: `http://localhost:3000/admin/facilities`
2. Scroll through the page and verify that all cards have an "Estimated hypertensive population:" subtitle AND that the subtitle shows either a comma-separated numeric value OR a "+ Add population" link
3. Click the "Edit district" button for one of the facility groups and edit the "Estimated hypertensive population" value
4. Save the changes and verify that the facility group's population value has been updated in the "Facilities" admin page
5. Find a different facility group and click the "Edit district" button. This time, delete the "Estimated hypertensive population" value and save the changes
6. Go back to the "Facilities" admin page and verify the facility group card shows the "+ Add population" link
7. Click the "+ Add population" link and set a population value
8. Go back to the "Facilities" admin page and verify the value is displayed in the facility group card

### CODE REVIEW

File: app/models/estimated_population.rb [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+        is_population_available = true
+      else
+        is_population_available = false
+        break
+      end
+    end
+    is_population_available
+  end
+
+  def update_state_population
```
Comment (rsanheim) [inline app/models/estimated_population.rb:32]: Lets discuss how to refactor this into a cleaner approach for a follow up PR.

File: app/models/estimated_population.rb [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+  validate :can_only_be_set_for_district_or_state
+  after_commit :update_state_population
+
+  def can_only_be_set_for_district_or_state
+    unless region.district_region? || region.state_region?
+      errors.add(:region, "can only set population for a district or a state")
+    end
+  end
+
+  def is_population_available_for_all_districts
```
Comment (rsanheim) [inline app/models/estimated_population.rb:18]: Is this used anywhere?

**Review by rsanheim — APPROVED** [review]
I pushed up a small change to keep the populate nil if the user doesn't set it - LGTM!

File: app/models/estimated_population.rb (hunk 2 above)
Comment (claudiovallejo) [reply app/models/estimated_population.rb:17]: @rsanheim this isn't used anywhere yet, but will use for the following PR when we show the population data in the Report "Details" pages

**Comment by claudiovallejo** [comment]
Will make these changes in PR #3107 :)

**Pull request closed without merging**